CHANGES.md

```diff
@@ -1,3 +1,6 @@
+- [IMPROVED] Improved performance by reducing the number of HTTP requests made
+  when instantiating `Database` objects
```

Comment:
Thanks @mikerhodes. It doesn't just apply to the `false` case though, the GET is also removed in the `true` case. How about:
```
- [IMPROVED] Remove unneeded GET request to the database when creating `Database` objects.
  Irregardless of the chosen create option it offered little protection:
  - with create `false` the database could be deleted or credentials changed during the use
    of the object.
  - with create `true` a database with the same name could be created between the GET and PUT.
```